resources/wiki-overrides.properties

```diff
@@ -62,13 +53,9 @@ vaddy-plugin=https://wiki.jenkins-ci.org/display/JENKINS/VAddy+Plugin
 violations=https://wiki.jenkins-ci.org/display/JENKINS/Violations
 xpdev=https://wiki.jenkins-ci.org/display/JENKINS/XP-Dev+Plugin
 
-# Workaround for INFRA-922
-absint-astree=https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910545
-
 github-scm-trait-notification-context=https://wiki.jenkins.io/display/JENKINS/Github+Custom+Notification+Context+SCM+Behaviour+Plugin
 ca-apm=https://wiki.jenkins.io/display/JENKINS/CA+APM+Plugin+1.x
 comments-remover=https://wiki.jenkins.io/display/JENKINS/XComment.io+-+Comments+Remover+Plugin
-qtest=https://github.com/jenkinsci/qtest-plugin
```

Comment:
https://github.com/jenkinsci/qtest-plugin/commit/eb33a469790bd2fb31ac78889fbf3b3f6b1bb283 references the same URL.